vm-setup/roles/v1aX_integration_test/tasks/boot_iso.yml

```diff
@@ -0,0 +1,48 @@
+---
+  - set_fact:
+      NUM_OF_CONTROLPLANE_REPLICAS: "{{ NUM_OF_CONTROLPLANE_REPLICAS|int }}"
+      NUMBER_OF_BMNUM_OF_WORKER_REPLICASH_MD: "{{ NUM_OF_WORKER_REPLICAS|int }}"
+
+  - name: Get workers bmh 
+    shell: kubectl get bmh -n "{{ NAMESPACE }}" | grep workers | cut -f1 -d' '
+    register: worker_bmh
+
+  - name: Update MD Metal3MachineTemplate to boot from ISO image located remotely
```

Comment:
This looks inaccurate, you are updating the BMH spec with the ISO image to boot from AFAIU

Reply:
Do we change the BMH spec or Metal3MachineTemplate. I think we should change the Metal3MachineTemplate.